src/core/src/pass/serialize.cpp

```diff
@@ -127,7 +128,8 @@ void ngfunction_2_ir(pugi::xml_node& node,
                      const std::map<std::string, ngraph::OpSet>& custom_opsets,
                      ConstantWriter& constant_write_handler,
                      int64_t version,
-                     bool deterministic);
+                     bool deterministic,
+                     bool ignore_names);
```

Comment:
I expect that you don't need to modify this file in case of custom pass.

Please revert all changes!

Reply:
I think so, will remove after finish a custom model hash.

Reply:
removed, thanks Ilya!